AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.html

```diff
@@ -109,6 +109,22 @@ <h2 class="govuk-heading-m">Add a participant</h2>
           <input id="phone" formControlName="phone" class="govuk-input govuk-input--width-20" type="text" />
         </div>
 
+        <div *ngIf="isInterpreter">
+          <div class="govuk-form-group" [ngClass]="{ 'govuk-form-group--error': interpreterForInvalid }">
```

Comment:
Recommend using `[class.` syntax for readability

Reply:
updated to [class.govuk-form-group--error]